packages/pwa/app/pages/product-detail/index.jsx

```diff
@@ -304,6 +304,11 @@ ProductDetail.getProps = async ({params, location, api}) => {
         })
     }
 
+    // Set the `cache-control` header values similar to those on the product-list.
+    if (res) {
+        res.set('Cache-Control', 'public, must-revalidate, max-age=900')
```

Comment:
https://pokeinthe.io/2021/09/13/cache-control-recommendations/

You can probably get away with just `max-age=900` here.